recipes/sortmerna/4.2.0/meta.yaml

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
Please remove these header lines as they don’t add any information not otherwise available. (Also in `build.sh`)

Reply:
done